playbooks/openshift/provision-openstack.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- hosts: localhost
+  roles:
+  - role: openstack-stack
+    stack_name: test-stack
+    dns_domain: "{{ openstack_dns_domain }}"
+    dns_nameservers: "{{ openstack_nameservers }}"
+    subnet_prefix: "{{ openstack_subnet_prefix }}"
+    ssh_public_key: "{{ openstack_ssh_public_key }}"
+    openstack_image: "{{ openstack_default_image_name }}"
+    etcd_flavor: "{{ openstack_default_flavor }}"
+    master_flavor: "{{ openstack_default_flavor }}"
+    node_flavor: "{{ openstack_default_flavor }}"
+    infra_flavor: "{{ openstack_default_flavor }}"
+    dns_flavor: "{{ openstack_default_flavor }}"
+    external_network: "{{ openstack_external_network_name }}"
+    num_etcd: 0
+    num_masters: 1
+    num_nodes: "{{ openstack_num_nodes }}"
+    num_infra: "{{ openstack_num_infra }}"
+    num_dns: 1
+
+- name: Refresh Server inventory
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  tasks:
+  - command: inventory/openstack.py --refresh --list
+  - meta: refresh_inventory
+
+- hosts: OSEv3:dns
+  tasks:
+  - name: waiting for server to come back
+    local_action: wait_for host={{ inventory_hostname }} state=started delay=30 timeout=300
```

Comment:
@sabre1041 we can't just include `all` here because we can't make the assumption that we're running in a clean OSP tenant.